compose/service.py

```diff
@@ -448,11 +479,26 @@ def recreate_container(self,
             else:
                 raise
```

Comment:
I'm pretty sure we don't need the try/except anymore (it can be just a plain `container.stop(timeout=timeout)`).

The api returns a 304 for already stopped containers which doesn't raise an error. Looks like it was added over a year ago b573b87a92d12bad82a41377e8cbaa891ac0dfb9. The API has been fixed since then.

If you don't feel comfortable removing the try/except in this PR, maybe just add a TODO for now and we can remove it later?